projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,585 @@
+	"advancements.projectexpansion.blue_fuel_block.description": "获得蓝色燃料块。",
+	"advancements.projectexpansion.cyan_fuel": "青色燃料",
+	"advancements.projectexpansion.cyan_fuel.description": "获得青色燃料。",
+	"advancements.projectexpansion.cyan_fuel_block": "青色燃料块",
+	"advancements.projectexpansion.cyan_fuel_block.description": "获得青色燃料块。",
+	"advancements.projectexpansion.green_fuel": "绿色燃料",
+	"advancements.projectexpansion.green_fuel.description": "获得绿色燃料。",
+	"advancements.projectexpansion.green_fuel_block": "绿色燃料块",
+	"advancements.projectexpansion.green_fuel_block.description": "获得绿色燃料块。",
+	"advancements.projectexpansion.yellow_fuel": "黄色燃料",
+	"advancements.projectexpansion.yellow_fuel.description": "获得黄色燃料。",
+	"advancements.projectexpansion.yellow_fuel_block": "黄色燃料块",
+	"advancements.projectexpansion.yellow_fuel_block.description": "获得黄色燃料块。",
+	"advancements.projectexpansion.lime_fuel": "浅绿燃料",
+	"advancements.projectexpansion.lime_fuel.description": "获得浅绿燃料。",
+	"advancements.projectexpansion.lime_fuel_block": "浅绿燃料块",
+	"advancements.projectexpansion.lime_fuel_block.description": "获得浅绿燃料块。",
+	"advancements.projectexpansion.orange_fuel": "橙色燃料",
+	"advancements.projectexpansion.orange_fuel.description": "获得橙色燃料。",
+	"advancements.projectexpansion.orange_fuel_block": "橙色燃料块",
+	"advancements.projectexpansion.orange_fuel_block.description": "获得橙色燃料块。",
+	"advancements.projectexpansion.white_fuel": "白色燃料",
+	"advancements.projectexpansion.white_fuel.description": "获得白色燃料。",
+	"advancements.projectexpansion.white_fuel_block": "白色燃料块",
+	"advancements.projectexpansion.white_fuel_block.description": "获得白色燃料块。",
+	"advancements.projectexpansion.fading_fuel": "渐隐燃料",
+	"advancements.projectexpansion.fading_fuel.description": "获得渐隐燃料。",
+	"advancements.projectexpansion.fading_fuel_block": "渐隐燃料块",
+	"advancements.projectexpansion.fading_fuel_block.description": "获得渐隐燃料块。",
+	"advancements.projectexpansion.magnum_star_omega": "更大的EMC电池？？",
+	"advancements.projectexpansion.magnum_star_omega.description": "没有人应该拥有如此多的EMC。",
+	"advancements.projectexpansion.colossal_star_omega": "你真的需要这么多EMC吗？",
+	"advancements.projectexpansion.colossal_star_omega.description": "这真的太荒谬了。",
+	"advancements.projectexpansion.final_star_shard": "你玩这个游戏太久了",
+	"advancements.projectexpansion.final_star_shard.description": "真的，去外面走走吧。",
+	"advancements.projectexpansion.transmutation_interface": "在你的电脑里进行转化？？",
+	"advancements.projectexpansion.transmutation_interface.description": "这简直是质的飞跃。",
+	"advancements.projectexpansion.infinite_steak": "无尽牛排！",
+	"advancements.projectexpansion.infinite_steak.description": "面包？那种东西留给平民吧。",
+	"advancements.projectexpansion.infinite_fuel": "这是星星吗？",
+	"advancements.projectexpansion.infinite_fuel.description": "还是说是个十字架？管它呢，丢进熔炉里看看它的威力吧。",
+	"advancements.projectexpansion.collector": "更多的太阳能",
+	"advancements.projectexpansion.collector.description": "现在荒谬的事情开始了。",
+	"advancements.projectexpansion.compressed_collector": "收集器压缩之后变得没用了...",
```

```suggestion
	"advancements.projectexpansion.compressed_collector": "收集器压缩之后变得没用了……",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。